### ISSUE TITLE
[feature suggestion] choose path for active .tox profile

### ISSUE THREAD

**Issue by tvimt** [tux3/qTox#2996]
Tried qtox portable (windows) to run it from a cloud storage so I can use the same profile on all my computers without exporting and importing. 
But it seems only the application becomes portable, the profile still gets created in and run from C:/Users/userxxx/AppData/Roaming/tox

It would be very handy with the posibility to choose the profile location/path yourself. No matter if it's windows, linux, portal or not.

**Comment by ProMcTagonist** [comment]
+1, Critical for mode to be useful.

**Comment by linux-modder** [comment]
You can currently.

Click on your avator ( you get a user profile dialog )


click on the  current  tox profile location it  then allows you to  declare a new location for your .tox file

**Comment by zetok** [comment]
@linux-modder no, it doesn't allow selecting the path where profile is located. It only opens current profile location.

---

@tvimt 
>Tried qtox portable (windows) to run it from a cloud storage so I can use the same profile on all my computers without exporting and importing. 

Just so you know, using "the cloud" is a bad idea. You might want to try using syncthing. With it, you would have whatever directories you want synced.

---
>It would be very handy with the posibility to choose the profile location/path yourself. No matter if it's windows, linux, portal or not. 

How would that be done?
Things to consider:
* a way to supply it before qTox starts for the first time
* preferably easy to use, so that everyone in need could enjoy the feature

**Comment by ProMcTagonist** [comment]
An options tab in the login window? That's been suggested before for things like making sure qTox never connects without your proxy even on fresh profiles.

**Comment by zetok** [comment]
>An options tab in the login window?

Perhaps that's the way to do it.
